server/src/test/scala/actors/ArrivalsRestorerSpec.scala

```diff
@@ -0,0 +1,64 @@
+package actors
+
+import drt.shared.PortCode
+import drt.shared.Terminals.{T1, T2}
+import drt.shared.api.Arrival
+import drt.shared.ArrivalsRestorer
+import org.specs2.mutable.Specification
+
+import scala.collection.mutable
+
+class ArrivalsRestorerSpec extends Specification {
+  def newRestorer = new ArrivalsRestorer[Arrival]
+
+  val arrival1: Arrival = ArrivalGenerator.arrival(iata = "BA0001", terminal = T1, schDt = "2021-05-01T10:20")
+  val arrival2: Arrival = ArrivalGenerator.arrival(iata = "BA0002", terminal = T2, schDt = "2021-05-02T07:40", origin = PortCode("JFK"))
+  val arrival3: Arrival = ArrivalGenerator.arrival(iata = "BA0002", terminal = T2, schDt = "2021-05-02T07:40", origin = PortCode("TFF"))
+
+  "Given one update and no removals " +
```

Comment:
there is some context missing here. What are we doing with updates and removals?